examples/Console/Examples.Console.csproj

```diff
@@ -6,12 +6,23 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Protobuf Include="..\GrpcService\Protos\greet.proto" GrpcServices="Client" />
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.3.0" />
+    <PackageReference Include="Google.Protobuf" Version="3.13.0" />
```

Comment:
Not blocking this PR, we might need to extract these versions to a common `*.props` file before they became unmanageable.

Reply:
I thought about that. Noticed that most if not all the examples were not doing that. I'd be happy to follow up with a clean up PR that does all the example projects.